src/seat.c

```diff
@@ -636,3 +643,18 @@ seat_reset_pressed(struct seat *seat)
 	seat->pressed.toplevel = NULL;
 	seat->pressed.resize_edges = 0;
 }
+
+void
+seat_outputs_changed(struct seat *seat)
```

Comment:
`seat_handle_output_layout_change()` might be a better name.

Reply:
What about `seat_output_layout_changed`, this matches the likes of `seat_reset_pressed` (https://github.com/labwc/labwc/blob/master/include/labwc.h#L462)